apps/web/cypress/tests/templates-store.spec.ts

```diff
@@ -0,0 +1,192 @@
+describe('Templates Store', function () {
+  const getTemplateDetails = (templateName: string): { name: string; iconName: string } => {
+    const regexResult = /^:.{1,}:/.exec(templateName);
+    let name = '';
+    let iconName = 'fa-solid fa-question';
+    if (regexResult !== null) {
+      name = templateName.replace(regexResult[0], '').trim();
+      iconName = regexResult[0].replace(/:/g, '').trim();
+    }
+
+    return { name, iconName: iconName };
+  };
+
+  beforeEach(function () {
+    cy.initializeSession({ noTemplates: true }).as('session');
+    indexedDB.deleteDatabase('localforage');
```

Comment:
initialize the session without the templates and clear the indexedDB (blueprints cache)